api/src/bump_test.go

```diff
@@ -0,0 +1,198 @@
+package main
+
+import (
+	"bytes"
+	"crypto/md5"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+)
+
+func startServer() {
+	path := "store.json"
+	err := os.Remove(path)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	go HandleRequests()
+}
+
+func sendInt(guildId int) (br BumpResponse) {
+	reqBody, _ := json.Marshal(map[string]int{
+		"guildId": guildId,
+	})
+
+	resp, _ := http.Post("http://localhost:8080/V1/bump", "application/json", bytes.NewBuffer(reqBody))
```

Comment:
The port "8080" should reflect the constant set in "[/api/src/main.go](https://github.com/Coding-Web-Community/CodingBump/blob/1e1261cbd1450db1d13e7ec5ee425a7c49dbd44e/api/src/main.go#L14)". If this becomes a distributed application, then "localhost" will become problematic, thus this should also be a constant.

https://github.com/Coding-Web-Community/CodingBump/blob/1e1261cbd1450db1d13e7ec5ee425a7c49dbd44e/api/src/main.go#L14

Reply:
Good call!
Changed the URL to a global variable too!